eclair-core/src/test/scala/fr/acinq/eclair/integration/basic/fixtures/MinimalNodeFixture.scala

```diff
@@ -185,8 +188,10 @@ object MinimalNodeFixture extends Assertions {
     watch1.replyTo ! WatchFundingDeeplyBuriedTriggered(blockHeight, txIndex, fundingTx)
     watch2.replyTo ! WatchFundingDeeplyBuriedTriggered(blockHeight, txIndex, fundingTx)
 
-    waitReady(node1, channelId)
-    waitReady(node2, channelId)
+    eventually {
+      assert(getChannelState(node1, channelId) == NORMAL)
+      assert(getChannelState(node2, channelId) == NORMAL)
+    }
```

Comment:
That doesn't do much since we are already in `NORMAL` when entering this method.

Reply:
Right, I see symmetry between the two methods but actually it doesn't make sense here.